lib/maid/rules.sample.rb

```diff
@@ -39,35 +41,31 @@
   end
 
   rule 'Mac OS X applications in zip files' do
-    found = dir('~/Downloads/*.zip').select { |path|
-      zipfile_contents(path).any? { |c| c.match(/\.app\/Contents\//) }
-    }
+    found = dir('~/Downloads/*.zip').select do |path|
+      zipfile_contents(path).any? { |c| c.match(%r{\.app/Contents/}) }
+    end
 
     trash(found)
   end
 
   rule 'Misc Screenshots' do
     dir('~/Desktop/Screen shot *').each do |path|
-      if 1.week.since?(accessed_at(path))
-        move(path, '~/Documents/Misc Screenshots/')
-      end
+      move(path, '~/Documents/Misc Screenshots/') if 1.week.since?(accessed_at(path))
     end
   end
 
   # NOTE: Currently, only Mac OS X supports `duration_s`.
   rule 'MP3s likely to be music' do
     dir('~/Downloads/*.mp3').each do |path|
-      if duration_s(path) > 30.0
-        move(path, '~/Music/iTunes/iTunes Media/Automatically Add to iTunes/')
-      end
+      move(path, '~/Music/iTunes/iTunes Media/Automatically Add to iTunes/') if duration_s(path) > 30.0
```

Comment:
Oof, is this even still supported? Isn't iTunes long gone?

I honestly don't know and I use a Mac at work.  I think they moved to some Apple Music thing.

Reply:
I'll leave this as is for now; will look into it if we get issues/reports about it being broken.